packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -222,3 +228,18 @@ class StyletronClient implements StandardEngine {
 }
 
 export default StyletronClient;
+
+function findSheetIndexWithMedia(children, media) {
```

Comment:
I refactored this logic in https://github.com/styletron/styletron/pull/335/commits/d1ab18a616cdb446571a78721515aff91168202e to avoid `Array.from` and `Array.prototype.findIndex` which would be reliant upon ES2015 polyfills (not merely Babel transpilation).